config/scgitx.yml

```diff
@@ -21,4 +21,8 @@ base_branch: master
 staging_branch: staging
 prototype_branch: prototype
 
+aggregate_branches:
+  - prototype
+  - staging
```

Comment:
@phene By 'immutably', I mean that the values assigned to the `staging_branch` and `prototype_branch` config keys are always included in the `#aggregate_branches` list (below); since these '(additional) aggregate_branches' values in this example config are the same as those (staging and prototype) branch values in this example config, they are potentially-confusingly ambiguous/conflicting.  So, I'm suggesting using something else (like 'qa' and/or 'legacy' or something) here as well, to disambiguate those.